corehq/apps/cloudcare/static/cloudcare/js/form_entry/webformsession.js

```diff
@@ -193,6 +222,7 @@ WebFormSession.prototype.handleSuccess = function(resp, callback) {
         self.lastRequestHandled = resp.seq_id;
 
         try {
+            self.displayInstanceXml(resp);
```

Comment:
can we put this after the callback call? if this fails then it'll actually block rendering